src/_guides/language/analysis-options.md

```diff
@@ -212,77 +258,105 @@ analyzer:
+
+To suppress more than one rule, use a comma-separated list:
+
+{% prettify dart %}
+// ignore_for_file: unused_import, invalid_assignment
+{% endprettify %}
+
+
+### Suppressing rules for a line of code
+
 To suppress a specific rule on a specific line of code,
 preceed that line with a comment using the following format:
 
 {% prettify dart %}
 // ignore: <linter rule>
 {% endprettify %}
 
-For example:
+Here's example of ignoring code that causes a runtime error,
+as you might do in a language test:
 
 {% prettify dart %}
 // ignore: invalid_assignment
 int x = '';
 {% endprettify %}
 
-If you want to suppress more than one rule, supply a comma-separated list.
-For example:
+To suppress more than one rule, supply a comma-separated list:
 
 {% prettify dart %}
 // ignore: invalid_assignment, const_initialized_with_non_constant_value
 const x = y;
 {% endprettify %}
 
-Alternatively, you can append the ignore rule to the line that it applies to.
-For example:
+Alternatively, append the ignore rule to the line that it applies to:
 
 {% prettify dart %}
 int x = ''; // ignore: invalid_assignment
 {% endprettify %}
 
-## Ignoring specific analysis rules
+## Customizing analysis rules
+
+Each [analyzer error code][analyzer error codes] and
+[linter rule][linter rules] has a default severity.
+You can use the analysis options file to change
+the severity of individual rules, or to always ignore some rules.
+
+The analyzer supports three severity levels:
+
+`info`
+: An informational message that doesn't cause analysis to fail.
+  Example: [`todo`][todo]
+
+`warning`
+: A warning that doesn't cause analysis to fail unless
+  the analyzer is configured to treat warnings as errors.
+  Example: Linter rules, such as [`avoid_as`][avoid_as]
+  **[PENDING: are all linter rules warnings by default?]**
+
+`error`
+: An error that causes analysis to fail.
+  Example: [`invalid_assignment`][invalid_assignment]
 
-Sometimes your code doesn't fit perfectly within the standard
-analysis guidelines, or violates a rule here or there, for
-reasons you'd rather not get into. You can ignore specific
-rules during analysis using the `errors:` field. List the
-rule, followed by `: ignore`. For example:
+
+
+### Ignoring rules
+
+You can ignore specific [analyzer error codes][] using the `errors:` field.
+List the rule, followed by <code>:&nbsp;ignore</code>. For example, the following
+analysis options file instructs the analysis tools to ignore the TODO rule:
 
 {% prettify yaml %}
 analyzer:
   errors:
     todo: ignore
 {% endprettify %}
 
-This analysis options file instructs the analysis tools to ignore
-the TODO rule.
-
-Alternatively, as of Dart 1.24 you can ignore a specific rule for a
-specific file using an `ignore_for_file` comment:
+**[PENDING: The wording implies that `ignore` works only for analyzer error codes,
+not for linter rules.
+Could you also use `ignore` with an included bunch of linter rules (e.g. pedantic)?]**
```

Comment:
Yes, it works with lint rules too.

Possible future: There's discussion that we might stop allowing users to ignore errors, but no decision yet.